themes/unfold/unfold.theme

```diff
@@ -0,0 +1,44 @@
+
+_lp_unfold_theme_activate() {
+    _lp_default_theme_activate
+}
+
+_lp_unfold_theme_directory() {
+    _lp_default_theme_directory
+}
+
+_lp_unfold_theme_prompt() {
+    _lp_default_theme_prompt_data
+
+    if [[ -f "${LP_PS1_FILE-}" ]]; then
+        # shellcheck source=liquid.ps1
+        source "$LP_PS1_FILE"
+    fi
```

Comment:
Why support this? If a user has this set, this theme is no different from the default theme. It might even be confusing for a user to enable this theme and see nothing change.

Reply:
For both questions: I don't really know. I just used the "default" theme functions and build up from there.

Reply:
Fair enough 😄 

For this question, I say remove this code. No need to support old style themes in this new thing.

For the other question, a new style theme is probably the right choice. So I guess forget I asked.

Reply:
✅ done.